src/clipboard.js

```diff
@@ -124,19 +142,4 @@ class Clipboard extends Emitter {
   }
 }
 
```

Comment:
also, any particular reason why we moved this method up?

Reply:
This function was used before it was initialized, so I moved to the top of the file.